assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -47,33 +41,17 @@ export default Component.extend(TextareaTextManipulation, ComposerUploadUppy, {
   emojiStore: service("emoji-store"),
   editingMessage: null,
   fullPage: false,
-  mediaOptimizationWorker: service(),
   onValueChange: null,
   previewing: false,
   showToolbar: false,
   timer: null,
   value: "",
+  inProgressUploads: null,
   composerFocusSelector: ".chat-composer-input",
-  useUploadPlaceholders: false,
-
-  // Composer Uppy values
-  ready: true,
-  composerEventPrefix: "chat-composer",
   canAttachUploads: or(
     "siteSettings.chat_allow_uploads",
     "chatChannel.isDirectMessageChannel"
   ),
-  composerModel: null,
-  composerModelContentKey: "value",
-  editorInputClass: ".chat-composer-input",
-  showCancelBtn: or("isUploading", "isProcessingUpload"),
-  uploadCancelled: false,
-  uploadProcessorActions: null,
-  uploadPreProcessors: null,
-  uploadMarkdownResolvers: null,
-  uploadType: "chat-composer",
-  uppyId: "chat-composer-uppy",
-  editorClass: alias("editorInputClass"),
 
```

Comment:
I dont see onValueChange initialized here

Reply:
It's passed in from chat-live-pane:

https://github.com/discourse/discourse-chat/blob/e6aab40cb0124097faae65dd5475699acf96be3d/assets/javascripts/discourse/templates/components/chat-live-pane.hbs#L127-L127